pallets/ds-accounts/src/mock.rs

```diff
@@ -109,6 +112,8 @@ impl Trait for Test {
     type AccountRole = u8;
     type Currency = pallet_balances::Module<Self>;
     type WeightInfo = ();
+    type MetaIPFS = Vec<u8>;        
+    type SerialNumber = String;    //not sure which type use here, for simplicity will be string
```

Comment:
tend to avoid String datatype since it  drags on havy utf8/bytes conversion code.

Reply:
swapped to Vec<u8>. 
utf8/bytes conversion is now only in tests.rs. Just to make tests easy to read.